src/core/ddsi/src/ddsi_discovery_spdp.c

```diff
@@ -371,22 +375,28 @@ static void ddsi_spdp_directed_xevent_cb (struct ddsi_domaingv *gv, struct ddsi_
 static void resched_spdp_broadcast (struct ddsi_xevent *ev, struct ddsi_participant *pp, ddsrt_mtime_t tnow)
 {
   /* schedule next when 80% of the interval has elapsed, or 2s
-   before the lease ends, whichever comes first (similar to PMD),
-   but never wait longer than spdp_interval */
+     before the lease ends, whichever comes first (similar to PMD),
+     but never wait longer than spdp_interval */
```

Comment:
With these code changes this code comment now applies to the `else` branch below, and 'interval' concerns the participant's lease duration. So may be better to either move it down or include the SPDP interval in the text.